src/main/webapp/script.js

```diff
@@ -73,19 +78,50 @@ function linkListElement(url) {
   return liElement;
 }
 
-/** TODO: Create Pagination for Dashboard*/
-
 /** Retrieves the previous page */
 function previous() {  // eslint-disable-line
-  /**   TODO */
```

Comment:
Maybe as a follow-up, you could try disabling/hiding the previous button if it's on the first page?

Reply:
The check method does this.

Reply:
Oh thanks - I missed that. Could you rename the method to something like #updatePageButtons()?